test/Test/Database/LSMTree/ModelIO/Class.hs

```diff
@@ -24,6 +24,8 @@ type IsSession :: ((Type -> Type) -> Type) -> Constraint
 class IsSession s where
     newSession :: IOLike m => m (s m)
 
+    closeSession :: IOLike m => s m -> m ()
+
```

Comment:
INFO: added this so we can properly clean up sessions after running statemachine tests